### PR TITLE
feat: organize CSS structure and implement product listing page

### LINKED ISSUE #3: Move CSS to it's own folder/file
We need to ensure the separation of concerns. As our code base grows it can become harder to separate CSS from HTML files.

We need to create a folder for CSS files and link them in the HTML.

### LINKED ISSUE #4: Pet-Shop products
When the user clicks the pet-shop card, he needs to see a page listing all the products in our application.

Let's create a page showing a list of 6 products with their prices:

Dog food: 100
Cat food: 50
Rat food: 10
Fish food: 1
Dog house: 200
Aquarium: 500

### PR DESCRIPTION
**Pull request by manoelmndf**
**Changes made**
1. A specific folder has been created for CSS files.  [close #3](https://github.com/ndvo/pet-shop-manoel/issues/3)
2. Implemented the product listing page when clicking on the “pet-shop” card. [close #4](https://github.com/ndvo/pet-shop-manoel/issues/4)

### CODE REVIEW

**Comment by ndvo** [comment]
we should avoid the "Implements..." part of the commit message.
It needs to be short and we end up wasting some characters with this.

So, instead of "Implementation of the product page and menu bar" we can use "Product page and menu bar". The same is true for other commits.

Can you use `git rebase` to fix all these commit titles?

**Comment by manoelmndf** [comment]
> There is a typo we need to fix, and we need to improve a little the commit messages (removing the "implements" part).

Thank you for the feedback. I have update all the commit titles to remove the text "implements"

File: index.html [hunk 1]
```diff
@@ -4,69 +4,66 @@
...
-            text-align: center;
-            padding: 1rem;
-        }
-    </style>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
 </head>
-<body>
-    <header>
+<<body class="d-flex flex-column min-vh-100">
```
[the remark this answers is not in the document]
Comment (manoelmndf) [reply index.html]: fixed typo in code

**Comment by ndvo** [comment]
Please, add links to the issues you are resolving in your PR description so that github will automatically close them upon merge:

```
close #3
close #4 
```

**Comment by manoelmndf** [comment]
> Please, add links to the issues you are resolving in your PR description so that github will automatically close them upon merge:
> 
> ```
> close #3
> close #4 
> ```

> [close #3](https://github.com/ndvo/pet-shop-manoel/issues/3)
>[close #4](https://github.com/ndvo/pet-shop-manoel/issues/4)

**Comment by manoelmndf** [comment]
> **Changes made**
> 
> 1. A specific folder has been created for CSS files.
> 2. Implemented the product listing page when clicking on the “pet-shop” card.

> [close #3](https://github.com/ndvo/pet-shop-manoel/issues/3)
>[close #4](https://github.com/ndvo/pet-shop-manoel/issues/4)

**Review by ndvo — APPROVED** [review]

**Pull request merged**